scoreboard.pl

```diff
@@ -1642,8 +1642,13 @@ ($$@)
 ## for now, implement it as a no-op which returns something which is undefined.
 sub GetVariable ($)
 {
-   my %a=();
-   return $a{'UNDEFINED'};
+    my $v = shift;
+    if ($v eq 'junit_xml_output') {
+        return 1;
```

Comment:
This is certainly useful for testing, but do you intend to merge this way?
If we want it supported in `scoreboard.pl` it could be controlled by a command-line option.

Reply:
If that makes more sense, sure. For now prettify.pm depends on this to return a boolean or undefined. We may need to change that, too.

Reply:
added option -u for generating _JUnit.xml files

Reply:
We set junit_xml_output in our builds to the file prefix `junit_xml`, this results in a `junit_xml_JUnit.xml` file, do you intend to change this behavior?

Reply:
This change only impacts the case where `scoreboard.pl` initiates parsing the plaintext.  Since the previous version of this function always returned false unconditionally, this shouldn't impact the case you're referring to.
It is mostly for testing since `scoreboard.pl` lets you run the prettify step outside of `autobuild.pl`

Reply:
Yes, and $junit_xml_output is default to 0. It's set to 1 only when -u is specified as in the test step 1.2 above.

Reply:
I haven't checked all code, so not sure what exactly changes but we set `$unit_xml_output` to a prefix, so it is not a bool in our case. 

I don't want to build scripts to add -u, the existence of the `junit_xml_output` as variable in the autobuild xml file should be enough to trigger the generation of the file. Not all build xml files do have this at our side, some have and some don't. The filename is fixed using the prefix so that jenkins can check for the file

Reply:
You do not need to update any scripts or config files.  Processing a log from `autobuild.pl` (which is the only thing that currently uses `junit_xml_output` will not be changed by this.